### ISSUE TITLE
Not passing args to RepeatingTimer

### ISSUE THREAD

**Issue by evgeniikozlov** [joshburnett/multitimer#1]
It seems that passing args to RepeatingTimer is missed.
multitimer.py, line 154:
```
        self._timer = RepeatingTimer(interval=self._interval, function=self._function,
                                     kwargs=self._kwargs, count=self._count,
                                     runonstart=self._runonstart)
```

**Comment by joshburnett** [comment]
Hmmm, looks like you are correct. I must have missed that argument when switching from params to args & kwargs in 0.2. I'll fix that.

**Comment by polvoazul** [comment]
:+1:

**Comment by Nxt-1** [comment]
@joshburnett I just ran into this issue as well. Any chance of this getting fixed and redistributed?

**Comment by joshburnett** [comment]
Fixed in v0.3, now released.
